test-app/app/Services/MyService.php

```diff
@@ -2,10 +2,12 @@
 
 namespace TestApp\Services;
 
+use Illuminate\Database\Eloquent\Collection;
+
 class MyService
 {
-    public function getData($orderBy): array
+    public function getData($orderBy): Collection
     {
-        return [];
+        return new Collection([]);
```

Comment:
Not an issue, but for test it's enough to return array - it is simpler. Were there any issues with it?

Reply:
Agree with you.